src/engine/strat_engine/backstore/metadata.rs

```diff
@@ -196,68 +231,74 @@ impl StaticHeader {
     }
 
     /// Try to find a valid StaticHeader on a device.
-    /// If there is no StaticHeader on the device, return None.
-    /// If there is a problem reading a header, return an error.
+    /// Return None if the static header's magic does not match for *both* copies.
```

Comment:
In final copy, method header comment should document repair work.

Reply:
@mulkieran I had repair notes in original header, @agrover wanted them removed...

Reply:
I just wanted the "we must" removed, those two words, in three instances...

Reply:
@agrover Ah, OK will do